classes/event/massaction_duplicated.php

```diff
@@ -0,0 +1,71 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace block_massaction\event;
+
+use core\event\base;
+
+/**
+ * The massaction_duplicated event class.
+ *
+ * @package     block_massaction
+ * @category    event
+ * @author      Tomo Tsuyuki <tomotsuyuki@catalyst-au.net>
+ * @copyright   2023 Catalyst IT
+ * @license     https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class massaction_duplicated extends base {
```

Comment:
We already have this event under massaction block. We don't really need to duplicate it in event name. How about renaming to **activities_duplicated** or **course_modules_duplicated**?

Reply:
I renamed it to "course_modules_duplicated".